src/cmdstan-guide/compiling_stan_programs.Rmd

```diff
@@ -159,7 +159,7 @@ and hardware used. The use of these flags does considerably slow down compilatio
 
 ### Optimizing by ignoring range checks
 
-When assigning or reading from with vectors, row_vectors, matrices or arrays using indexing, Stan performs cheks that
+When assigning or reading from with vectors, row_vectors, matrices or arrays using indexing, Stan performs checks that
 a supplied index is valid (not out of range). These check avoids segmentation faults and other runtime errors
```

Comment:
lines163-164 - "checks" - better yet:
Stan checks that a supplied index is valid (not out of range), which avoids segmentation faults and other difficult-to-debug runtime errors.